### PR TITLE
[spotify-web-playback-sdk] Reflect new changes

### PR DESCRIPTION
**Pull request by NeoLegends**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).

Select one of these and delete the others:

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://beta.developer.spotify.com/documentation/web-playback-sdk/reference/#api-spotify-player-addlistener
- [X] Increase the version number in the header if appropriate.

----

Spotify has added new methods for adding and removing event handlers to their web playback sdk. `addListener` behaves exactly like `on`. There has been no pendant for `removeListener` before. This has been added in.

### CODE REVIEW

**Comment by andy-ms** [comment]
This PR has been open and unchanged 5 days without signoff or complaint. This will be merged by a maintainer soon if there are no objections.

**Review by mraerino — APPROVED** [review]

**Pull request merged**
